x-pack/plugins/security_solution/public/flyout/left/components/prevalence_details_count_cell.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { VFC } from 'react';
+import React from 'react';
+import { EuiLoadingSpinner } from '@elastic/eui';
+import {
+  PREVALENCE_DETAILS_COUNT_CELL_LOADING_TEST_ID,
+  PREVALENCE_DETAILS_COUNT_CELL_VALUE_TEST_ID,
+} from './test_ids';
+import type { EventType } from '../../shared/hooks/use_fetch_field_value_pair_by_event_type';
+import { useFetchFieldValuePairByEventType } from '../../shared/hooks/use_fetch_field_value_pair_by_event_type';
+import { TimelineId } from '../../../../common/types';
+
+export interface PrevalenceDetailsCountCellProps {
+  /**
+   * The field name
+   * */
+  field: string;
+  /**
+   * The field values
+   */
+  values: string[];
+  /**
+   * The scope id
+   */
+  scopeId: string;
+  /**
+   * Limit the search to include or exclude a specific value for the event.kind field
+   * (alert, asset, enrichment, event, metric, state, pipeline_error, signal)
+   */
+  type: EventType;
+}
+
+/**
+ * Component displaying a value in many PrevalenceDetails table cells. It is used for the third and fourth columns,
+ * which display the number of alerts and documents for a given field/value pair.
+ * For the doc columns, type should "signal" for its eventKind property, and exclude set to true.
+ * For the alert columns, type should have "signal" for its eventKind property, and include should be true.
+ */
+export const PrevalenceDetailsCountCell: VFC<PrevalenceDetailsCountCellProps> = ({
+  field,
+  values,
+  scopeId,
+  type,
+}) => {
+  const { loading, error, count } = useFetchFieldValuePairByEventType({
+    field,
+    values,
+    isActiveTimelines: scopeId === TimelineId.active,
+    type,
+  });
+
+  if (loading) {
+    return <EuiLoadingSpinner data-test-subj={PREVALENCE_DETAILS_COUNT_CELL_LOADING_TEST_ID} />;
+  }
+
+  if (error) {
+    return null;
```

Comment:
actually maybe we should return something here?:)

Reply:
I originally thought that having an empty cell would be ok, but I agree that showing some sort of error is better. I decided to return an error icon, let me know what you think... did you have something else in mind?
![Screenshot 2023-06-07 at 9 47 29 AM](https://github.com/elastic/kibana/assets/17276605/71316eb6-ff07-45b8-9cac-f3388bbde227)

Reply:
this looks okay to me, but maybe @ferenrigue  has some better idea:)